checks/system/integration/automated_tests_rfm.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import yaml
+from utils import *
+from constants import *
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+import glob
+import json
+
+# --------------------------------------------------------------------------- #
+#                READ THE CONFIGURATION FROM THE YAML FILES
+#                This part should be modified for the final 
+#                structure of the V-Clusters config files
+# --------------------------------------------------------------------------- 
+
+# Read the extracted info from the json file
+with open(json_file_path+'test_data.json', 'r') as json_file:
+    config_yaml_data = json.load(json_file)
+
+test_info = {}
+# Check for mount points to be checked
+try:
+    mount_info = config_yaml_data[MOUNT_VARS]
+    test_info.update({MOUNT_VARS : mount_info})
+except:
+    mount_info = [None]
+
+# Check for pkg installations to be checked
+try:
+    tools_info = config_yaml_data[TOOLS_VARS]
+    test_info.update({TOOLS_VARS : tools_info})
+except:
+    tools_info = [None]
+
+# Check for environment variables to be checked
+try:
+    envs_info = config_yaml_data[ENV_VARS]
+    test_info.update({ENV_VARS : envs_info})
+except:
+    envs_info = [None]
+
+# --------------------------------------------------------------------------- #
+#                PERFORM INTEGRATION CHECKS
+# --------------------------------------------------------------------------- #
+
+@rfm.simple_test
+class MountTest(rfm.RunOnlyRegressionTest):
+
+    if mount_info[0] is not None:
+        mount_info_par = []
+        for mount_i in mount_info:
+            mount_info_par.append((mount_i["mount_point"], mount_i["fstype"]))
+        mount_info = parameter(mount_info_par)
+    
+    descr = 'Test mount points in the system'
+    # For now
+    valid_systems = ["-remote"]
```

Comment:
You could directly set valid_systems to an empty list when `mount_info` is `None`. Mostly to avoid adding more lines to the output (skipping test ...)